cluster/log-dump/log-dump.sh

```diff
@@ -482,14 +483,13 @@ function dump_nodes_with_logexporter() {
       proc=${max_dump_processes}
       wait
     fi
-  done
+  done < <("${KUBECTL}" get pods -n "${logexporter_namespace}" -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.nodeName}{"\n"}{end}')
```

Comment:
shellcheck detects SC2030 and SC2031. To avoid that, I remove pipe.
```
In ./cluster/log-dump/log-dump.sh line 480:
    proc=$((proc - 1))
    ^--^ SC2030: Modification of proc is local (to subshell caused by pipeline).
```
```
In ./cluster/log-dump/log-dump.sh line 487:
  if [[ proc -gt 0 && proc -lt ${max_dump_processes} ]]; then
        ^--^ SC2031: proc was modified in a subshell. That change might be lost.
                      ^--^ SC2031: proc was modified in a subshell. That change might be lost.
```